tagmanager/src/tm_source_file.c

```diff
@@ -109,6 +109,35 @@ gchar *tm_get_real_path(const gchar *file_name)
 	return NULL;
 }
 
+/* add argument list of __init__() Python methods to the class tag */
+static void update_python_arglist(const TMTag *tag)
+{
+	guint i;
+	const char *parent_tag_name;
+
+	if (tag->type != tm_tag_method_t || tag->scope == NULL ||
+		g_strcmp0(tag->name, "__init__") != 0)
+		return;
+
+	parent_tag_name = strrchr(tag->scope, '.');
```

Comment:
note: doesn't support the `/` separator previous code did, but our parser doesn't have it anyway.
Also, it would only concern classes defined inside a function or method, which is less common.